### PR TITLE
Add book command for idENTify

### PR DESCRIPTION
**Pull request by JovonLim**

### CODE REVIEW

**Review by Guanzhou03 — APPROVED** [review]

**Review by pwjj2000 — APPROVED** [review]
LGTM!

**Review by NicholasTYD — APPROVED** [review]
LGTM

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "name": "Hans Muster",
     "phone": "9482424",
     "email": "invalid@email!3e",
-    "address": "4th street"
+    "address": "4th street",
+    "appointments" : []
```
Comment (Guanzhou03) [inline src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:7]: Is there a difference between [ ] and []?

File: src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java [hunk 2]
```diff
@@ -28,6 +29,7 @@ public class JsonAdaptedPersonTest {
     private static final String VALID_PHONE = BENSON.getPhone().toString();
     private static final String VALID_EMAIL = BENSON.getEmail().toString();
     private static final String VALID_ADDRESS = BENSON.getAddress().toString();
+    private static final List<Appointment> VALID_APPOINTMENTS = BENSON.getAppointments();
```
Comment (Guanzhou03) [inline src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java:32]: Should we have an invalid appointment?

**Review by Guanzhou03 — COMMENTED** [review]
Implementation of book command looks solid, but I wonder if there should be tests for invalid appointments in the JsonAdapatedPersonTest.java

**Pull request merged**